src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -100,14 +105,7 @@ static ConnectionProvider elastic(String name) {
 	 * {@link Connection}
 	 */
 	static ConnectionProvider elastic(String name, @Nullable Duration maxIdleTime, @Nullable Duration maxLifeTime) {
```

Comment:
The builder should be on ConnectionProvider level because we otherwise if a new property has to be added we need to add additional method like this